roles/openstack_health_monitor/README.rst

```diff
@@ -114,9 +114,14 @@ The container image to use.
 .. zuul:rolevar:: openstack_health_monitor_container_name
    :default: openstack_health_monitor
 
-Name of the container in which Openstack-health-monitor will run.
+Name of the container.
+
+.. zuul:rolevar:: openstack_health_monitor_service_name
+   :default: openstack_health_monitor
+
+Name of the service.
 
 .. zuul:rolevar:: openstack_health_monitor_network
    :default: 172.31.100.160/28
 
-The network to use for the Openstack-health-monitor container.
\ No newline at end of file
+Name of the network.
```

Comment:
An address is not a name IMO. Also what is "the network"? Why not just leave the old description in place?